metaflow/plugins/__init__.py

```diff
@@ -59,17 +59,22 @@ def _merge_lists(base, overrides, attr):
 from .conda.conda_step_decorator import CondaStepDecorator
 from .kfp.kfp_decorator import KfpInternalDecorator
 from .kfp.pytorch_distributed_decorator import PyTorchDistributedDecorator
-
-STEP_DECORATORS = _merge_lists([CatchDecorator,
-                                TimeoutDecorator,
-                                EnvironmentDecorator,
-                                ResourcesDecorator,
-                                RetryDecorator,
-                                BatchDecorator,
-                                StepFunctionsInternalDecorator,
-                                CondaStepDecorator,
-                                PyTorchDistributedDecorator,
-                                KfpInternalDecorator], ext_plugins.STEP_DECORATORS, 'name')
+from .kfp.kfp_pod_decorator import (PodAnnotationDecorator, PodLabelDecorator)
+
+STEP_DECORATORS = _merge_lists([
+    CatchDecorator,
+    TimeoutDecorator,
+    EnvironmentDecorator,
+    ResourcesDecorator,
+    RetryDecorator,
+    BatchDecorator,
+    StepFunctionsInternalDecorator,
+    CondaStepDecorator,
+    PyTorchDistributedDecorator,
+    KfpInternalDecorator,
+    PodAnnotationDecorator,
+    PodLabelDecorator,
+], ext_plugins.STEP_DECORATORS, 'name')
```

Comment:
STEP_DECORATORS aren't used anywhere?

Reply:
oops, never-mind, we can revert this if we use the `@kfp` decorator instead and not introduce a new one. thanks!